src/js/jsx/search/SearchBar.jsx

```diff
@@ -400,6 +400,19 @@ define(function (require, exports, module) {
         /** @ignore */
         _handleDialogClick: function (event) {
             this.refs.datalist.removeAutofillSuggestion();
+            
+            // If string, it means no items match user inputted search query
+            var filters = this.state.filter,
```

Comment:
This is called `filters` because there can be more than one filter added at a time, i.e. "Layers" > "Text Layers"